ui-src/src/cells/holo-chat/actions.ts

```diff
@@ -1,4 +1,6 @@
 export const MESSAGESLIST = 'messagesList'
+export const CHANNELSLIST = 'channelsList'
```

Comment:
I have some cool stuff in another branch that lets us not use these constants and rely on typing instead. Will merge in asap